README.md

```diff
@@ -21,6 +21,11 @@ This repository contains examples built with PSPDFKit. These examples are not of
+- [Programmatically Create a Comment Thread](https://pspdfkit.com/playground?p=eyJ2IjoxLCJzZXR0aW5ncyI6eyJmaWxlTmFtZSI6ImJhc2ljLnBkZiIsImp3dCI6eyJwZXJtaXNzaW9ucyI6WyJyZWFkLWRvY3VtZW50Iiwid3JpdGUiXSwidXNlcklkIjoicmFuZG9tbEUxUXUwYzdjVCIsImxheWVyIjoicmFuZG9ta1FnUzRqOEpySCJ9fSwianMiOiIvL3NldCB1c2VyIG5hbWUgZnJvbSBhIGxvZ2luIG9yIGF1dGhlbnRpY2F0aW9uXG5jb25zdCB1c2VybmFtZT0nT21hcic7UFNQREZLaXQubG9hZCh7Li4uYmFzZU9wdGlvbnMsdGhlbWU6UFNQREZLaXQuVGhlbWUuREFSSyx0b29sYmFySXRlbXM6W3t0eXBlOidjb21tZW50J31dfSkudGhlbigoaW5zdGFuY2U9Pntjb25zdCBhbm5vdGF0aW9uPW5ldyBQU1BERktpdC5Bbm5vdGF0aW9ucy5Db21tZW50TWFya2VyQW5ub3RhdGlvbih7aWQ6J3Rlc3QnLHBhZ2VJbmRleDowLGJvdW5kaW5nQm94Om5ldyBQU1BERktpdC5HZW9tZXRyeS5SZWN0KHt0b3A6MTAwLGxlZnQ6MTAwLHdpZHRoOjEwLGhlaWdodDoxMH0pLGN1c3RvbURhdGE6e2NpcmNsZUlkOidteS1jaXJjbGUnfX0pLGNvbW1lbnQ9bmV3IFBTUERGS2l0LkNvbW1lbnQoe2lkOidjb21tZW50SWQnLHJvb3RJZDondGVzdCcscGFnZUluZGV4OjAsdGV4dDp7Zm9ybWF0OidwbGFpbicsdmFsdWU6J0hlbGxvJ319KTtpbnN0YW5jZS5zZXRBbm5vdGF0aW9uQ3JlYXRvck5hbWUoJ09tYXInKSxpbnN0YW5jZS5jcmVhdGUoW2Fubm90YXRpb24sY29tbWVudF0pLGluc3RhbmNlLmFkZEV2ZW50TGlzdGVuZXIoJ2Fubm90YXRpb25zLmNyZWF0ZScsKGFubm90YXRpb25zPT57Y29uc29sZS5sb2coYW5ub3RhdGlvbnMuZ2V0KDApLnRvSlMoKSl9KSl9KSk7IiwiY3NzIjoiLyogQWRkIHlvdXIgQ1NTIGhlcmUgKi9cbiJ9)
+- [Generate Bates with the page info dimensions](https://pspdfkit.com/playground?p=eyJ2IjoxLCJzZXR0aW5ncyI6eyJmaWxlTmFtZSI6ImJhc2ljLnBkZiIsImp3dCI6eyJwZXJtaXNzaW9ucyI6WyJyZWFkLWRvY3VtZW50Iiwid3JpdGUiXSwidXNlcklkIjoicmFuZG9tbEUxUXUwYzdjVCIsImxheWVyIjoicmFuZG9ta1FnUzRqOEpySCJ9fSwianMiOiJQU1BERktpdC5sb2FkKHsuLi5iYXNlT3B0aW9ucyx0aGVtZTpQU1BERktpdC5UaGVtZS5EQVJLfSkudGhlbigoYXN5bmMgaW5zdGFuY2U9Pntcbi8vZ2VuZXJhdGluZyB0aGUgYmF0ZXMgdXNpbmcgdGhlIHBhZ2UgaW5mbyBkaW1lbnNpb25zXG5sZXQgYmF0ZXM9KGF3YWl0IFByb21pc2UuYWxsKEFycmF5LmZyb20oe2xlbmd0aDppbnN0YW5jZS50b3RhbFBhZ2VDb3VudH0pLm1hcCgoKF8scGFnZUluZGV4KT0%252BaW5zdGFuY2UucGFnZUluZm9Gb3JJbmRleChwYWdlSW5kZXgpKSkpKS5yZWR1Y2UoKChhY2MscGFnZWluZm8pPT57Y29uc3R7aW5kZXg6aW5kZXgscmF3UGRmQm94ZXM6cmF3UGRmQm94ZXN9PXBhZ2VpbmZvLHtjcm9wQm94OmNyb3BCb3h9PXJhd1BkZkJveGVzLFtsZWZ0LHRvcCx3aWR0aCxoZWlnaHRdPWNyb3BCb3gsdmFsdWU9UFNQREZLaXQuZ2VuZXJhdGVJbnN0YW50SWQoKTtyZXR1cm4gYWNjLmNvbmNhdChuZXcgUFNQREZLaXQuQW5ub3RhdGlvbnMuVGV4dEFubm90YXRpb24oe3BhZ2VJbmRleDppbmRleCx0ZXh0Ontmb3JtYXQ6J3BsYWluJyx2YWx1ZTp2YWx1ZS50b1N0cmluZygpfSxmb250OidIZWx2ZXRpY2EnLGZvbnRTaXplOjExLGlzQm9sZDohMCxob3Jpem9udGFsQWxpZ246J2NlbnRlcicsYm91bmRpbmdCb3g6bmV3IFBTUERGS2l0Lkdlb21ldHJ5LlJlY3Qoe2xlZnQ6d2lkdGgtMTEqdmFsdWUudG9TdHJpbmcoKS5sZW5ndGgsdG9wOjIwLHdpZHRoOjExKnZhbHVlLnRvU3RyaW5nKCkubGVuZ3RoLGhlaWdodDo0MH0pLGZvbnRDb2xvcjpQU1BERktpdC5Db2xvci5SRUR9KSl9KSxbXSk7YXdhaXQgaW5zdGFuY2UuY3JlYXRlKGJhdGVzKSxhd2FpdCBpbnN0YW5jZS5hcHBseU9wZXJhdGlvbnMoW3t0eXBlOidmbGF0dGVuQW5ub3RhdGlvbnMnfV0pfSkpOyIsIm1vZGUiOiJzdGFuZGFsb25lIiwiY3NzIjoiLyogQWRkIHlvdXIgQ1NTIGhlcmUgKi9cblx0In0%253D)

```

Comment:
This we can also skip as it's already being worked on internally: https://pspdfkit.com/guides/web/editor/add-bates-number/